server/src/main/resources/db/migration/V1_352__add_change_password_field_to_candidate.sql

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2025 Talent Catalog.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation, either version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see https://www.gnu.org/licenses/.
+ */
+
```

Comment:
Flyway's should not have the copyright notice. The reason is that it is messy to change Flyway files once they have been processed - so having copyright notices which are typically updated each year doesn't make sense on Flyway files.

Reply:
done